tests/pytorch/test_pytorch_autolog.py

```diff
@@ -112,13 +112,13 @@ def pytorch_model_with_callback(patience):
     )
 
     with TempDir() as tmp:
+        kwargs = (
+            {"dirpath": tmp.path()}
+            if LooseVersion(pl.__version__) >= LooseVersion("1.2.0")
+            else {"filepath": tmp.path()}
+        )
```

Comment:
Fix for this error:

https://github.com/mlflow/mlflow/runs/1880410847?check_suite_focus=true#step:8:2149